pkg/webhook/resources/storageclass/validator.go

```diff
@@ -269,3 +309,7 @@ func (v *storageClassValidator) validateReservedStorageClass(sc *storagev1.Stora
 
 	return nil
 }
+
+func (v *storageClassValidator) getEncryptionSecret(sc *storagev1.StorageClass) (secretName string, secretNamespace string) {
+	return sc.Parameters[pairs[0][0]], sc.Parameters[pairs[0][1]]
+}
```

Comment:
If we take this function alone, it doesn't check the length of pairs, which I consider a risk probably. What if combine this with the function `validateEncryptionParams()`? In the function `validateEncryptionParams()` it can still return the related secret.

Reply:
I just remove it.